src/content/foundation/index.md

```diff
@@ -31,11 +31,9 @@ Learn more at [devcon.org](https://devcon.org/), check out the [Devcon Blog](htt
 
 ### Fellowship Program {#fellowship-program}
 
-The [Ethereum Foundation Fellowship Program](https://fellowship.ethereum.foundation/) is a pilot project that focuses on seeking opportunities in emerging economies. Developing countries today account for over 80% of the world’s population. Amongst about 400,000 babies born every day around the world, more than 360,000 are born in developing countries. People in developing nations represent not only our future as a people, but they may also represent the next billion active users of Ethereum.
+The [Ethereum Foundation Fellowship Program](https://fellowship.ethereum.foundation/) is an initiative to help address gaps in representation across cultures, nationalities, and economic classes. The Fellowship Program is about bridging these gaps by identifying and supporting unique and talented individuals helping to enable Ethereum’s relevance and breaking down barriers to entry for those underrepresented people and communities who will become the future of web3.
 
-The Fellowship Program provides the EF an opportunity to work with change-makers, whose projects have high potential to create impact in emerging economies by using Ethereum. The program features a unique experience that includes mentorship, expertise and access to leaders in the Ethereum ecosystem.
-
-Learn more at [fellowship.ethereum.foundation](https://fellowship.ethereum.foundation/) or check out the [recent blog post on the Fellowship Program](https://blog.ethereum.org/2021/05/07/ethereum-for-the-next-billion/).
+Learn more at [fellowship.ethereum.foundation](https://fellowship.ethereum.foundation/).
```

Comment:
Giving the largest possible surface context for links is generally better for accessibility :-)

```suggestion
[Learn more at fellowship.ethereum.foundation](https://fellowship.ethereum.foundation/).
```